src/indexer/types.ts

```diff
@@ -101,6 +101,7 @@ export type RuneSpentUtxoBalance = RuneUtxoBalance & { spentTxid: string };
 export type RuneUtxoBalance = {
   txid: string;
   vout: number;
+  value?: number;
```

Comment:
This shouldn't be optional, there is always a value associated. Also, please rename to `satValue`, just to further disambiguate from `amount` that represents the rune count.

Reply:
nice suggestion